src/main/java/org/sciborgs1155/robot/feeder/Feeder.java

```diff
@@ -0,0 +1,27 @@
+package org.sciborgs1155.robot.feeder;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import monologue.Logged;
+import org.sciborgs1155.robot.Robot;
+
+public class Feeder extends SubsystemBase implements AutoCloseable, Logged {
```

Comment:
### Closed Loop Control on Feeder
This means `FeederIO` should provide a method for getting velocity, and `RealFeeder` should configure the spar appropriately.
There should be feedback feedforward in the `Feeder` subsystem, and sysid routines logged to smartdashboard